rust/processor/src/db/common/models/transaction_metadata_model/parquet_write_set_size_info.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::bq_analytics::generic_parquet_processor::{HasVersion, NamedTable};
+use allocative_derive::Allocative;
+use aptos_protos::transaction::v1::WriteOpSizeInfo;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct WriteSetSize {
+    pub txn_version: i64,
+    pub index: i64,
+    pub key_bytes: i64,
+    pub value_bytes: i64,
+}
```

Comment:
should there also be a `total_bytes` here (since you split the tag size vs total in events and this would be key/value/total?)

Reply:
looking into the proto, not sure what's the context behind the events spliting into tag and total. but we can add total here if that helps with query